content/pages/protocols/s3.html

```diff
@@ -99,7 +99,7 @@ <h3>Generic connection profiles for third party providers</h3>
                             </li>
                             <li>
                                 <span class="glyphicon glyphicon-download-alt" aria-hidden="true"></span>
-                                <a href="https://svn.cyberduck.io/trunk/profiles/S3%20(Credentials%20from%20AWS%20Security%20Token%20Service).cyberduckprofile">Download</a>
+                                <a href="https://profiles.cyberduck.io/S3%20(Credentials%20from%20AWS%20Security%20Token%20Service).cyberduckprofile">Download</a>
```

Comment:
I mentioned this in https://github.com/iterate-ch/cyberduck/issues/15795, but this profile appears to have been removed. Perhaps https://profiles.cyberduck.io/S3%20(Credentials%20from%20AWS%20Command%20Line%20Interface).cyberduckprofile should be mentioned and referenced instead or the section be removed. Should I open another issue?